### PR TITLE
uw_priority_queue: Exclude UW Housing residents from surveillance

### PR DESCRIPTION
**Pull request by jccraft-uw**

### CODE REVIEW

**Comment by jccraft-uw** [comment]
[Don't include UW Housing PTs when inviting for Surveillance](https://trello.com/c/QihQXqoz/1003-dont-include-uw-housing-pts-when-inviting-for-surveillance)

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -3338,6 +3338,8 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         -- Filter to participants who come to campus at least 2x a week
         -- OR who are Greek members
         where (on_campus_2x_week or uw_greek_member)
+        -- Filter to participants who are not UW Housing residents because they already get invited weekly
+        and uw_housing_group is null
```
Comment (kschwabefry) [inline schema/deploy/shipping/views.sql:3342]: Basing on standup discussion this morning, would the study team want us to lengthen the 3 day invitation interval below more broadly? The participant feedback of too-frequent invitation might apply to other participants getting invited for surveilance testing, not just UW housing residents

Comment (jccraft-uw) [reply schema/deploy/shipping/views.sql:3342]: The clinical team pushed for the 3 day interval between Surveillance invites. Originally, we had 21 days. Then 14 days. And we ended up running out of PTs to invite. The clinical team asked us to reduce this to 3 days to keep the participant pool flowing.

The commit message for 6c4f91a3bf30e047f0e89a852b269c164e3b14b6 was:
Reduce time between surveillance invites for UW queue

For the UW priority queue view, reduce the time between invitations
from 14 days to 3 days. With 14 days, we end up with days
when we run out of participants to invite for testing. 3 days is a balance.
With 1 day between invitations we would end up nagging participants too
soon.

Comment (kschwabefry) [reply schema/deploy/shipping/views.sql:3342]: Hmm, well, if they want this change, we can make it.

Comment (kschwabefry) [reply schema/deploy/shipping/views.sql:3342]: Alright. Thanks Chris.

**Review by kschwabefry — APPROVED** [review]

**Comment by jccraft-uw** [comment]
Deployed

**Comment by kschwabefry** [comment]
[Update reportable_conditions.py to send Apple and Schools positives to specific Slack channels](https://trello.com/c/4TcMuFGv/970-update-reportableconditionspy-to-send-apple-and-schools-positives-to-specific-slack-channels)

**Pull request merged**